linera-storage/src/lib.rs

```diff
@@ -473,4 +473,8 @@ where
     async fn get_blob(&self, blob_id: BlobId) -> Result<Blob, ExecutionError> {
         Ok(self.storage.read_blob(blob_id).await?)
     }
+
+    async fn assert_blob_exists(&self, blob_id: BlobId) -> Result<(), ExecutionError> {
```

Comment:
On the storage side I would make it return a `bool` and call it `contains_blob`, just like we do for certificate values.